pwndbg/commands/addressify.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from typing import Any
+
+import gdb
+
+import pwndbg.color.message as M
+import pwndbg.commands
+
+
+class AddressifyFunction(gdb.Function):
```

Comment:
btw, you can use existing decorator which extends gdb function. See https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/gdblib/functions.py